### DISCUSSION TITLE
problem zooming next/Image on IOS mobile web

### DISCUSSION THREAD

**Discussion by kimjy0301** [vercel/next.js#34118] [Help]
I implement the zoom function by enclosing the next/Image tag with a reaction-zoom-pan-pinch. 

It works fine on Android OS, but when expanding on the iPhone mobile web, the image looks broken.

The layout of the next/Image tag is intrinsic, and width and height are specified (2479x3508).
Is there a way to accurately see the image even if the user zooms in?

* ps. I tried to double the size of Div container and use useEffect hook to reduce it in half, but it didn't work normally.

this is my site url : [https://swmusic.vercel.app/catalog/1/64 ](https://swmusic.vercel.app/catalog/1/64 ) 

😭 😭 😭 😭 😭 😭 😭 😭 😭 😭 😭 😭 😭

**Comment by alex289** [comment]
I'm using an Iphone 12 and the image on your page looks fine to me. I can zoom in and out without any problem. Could it be your phone causing this?

**Reply by kimjy0301** [reply]
What I tested was iPhone Se2. Perhaps because the resolution of se2 was low, the figures in the drawing were not well seen when the image was zoomed in.

**Comment by hemmachat** [comment]
Hi @kimjy0301, I am trying to do the same image zooming using "next/image" component too. Please let me know how you leverage the full features of "next/image" with the zooming capability. Thanks

**Comment by thomasvillaverde** [comment]
Same problem with swiper js and next/image

**Reply by alagh697** [reply]
Hi its caused by the loop: if set to true it cause the problem.
